### PR TITLE
HD-1240: Build 1.5.2

### PR DESCRIPTION
**Pull request by sookim-altiscale**
RPM build for flume-1.5.2

### CODE REVIEW

File: scripts/justbuild.sh [hunk 1]
```diff
@@ -1,3 +1,17 @@
 #!/bin/sh -ex
-mkdir -p ${WORKSPACE}/.m2
+
+# We are installing untrusted code just to build hive for now.
```
Comment (raviprak-altiscale) [inline scripts/justbuild.sh]: Modify comment.

File: scripts/justbuild.sh [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+# Install these untrusted artifacts in our local maven cache so that we can build flume.
+mvn install:install-file -Dfile=ua-parser-1.3.0.jar -DgroupId=ua_parser -DartifactId=ua-parser -Dpackaging=jar -Dversion=1.3.0
+
+# Build flume
 mvn clean install -DskipTests
+
+# Discover the path of the local maven cache
+MAVEN_LOCAL_REPO=`mvn help:evaluate -Dexpression=settings.localRepository | egrep -v '[INFO]|Download'`
+
+# Remove the untrusted artifacts from this node's local maven repo soon as we have built hive
```
Comment (raviprak-altiscale) [inline scripts/justbuild.sh]: Here too

File: scripts/justinstall.sh [hunk 3]
```diff
@@ -1,20 +1,25 @@
...
+export RPM_NAME=`echo alti-flume-${FLUME_VERSION}`
+echo "Building Flume Version RPM ${RPM_NAME} with RPM version ${ALTISCALE_RELEASE}-${DATE_STRING}"
+
 cd ${RPM_DIR}
 fpm --verbose \
---maintainer ops@verticloud.com \
---vendor VertiCloud \
+--maintainer support@altiscale.com \
+--vendor Altiscale \
 --provides ${RPM_NAME} \
```
Comment (raviprak-altiscale) [inline scripts/justinstall.sh:20]: Could you please add URL?

Comment (sookim-altiscale) [inline scripts/justinstall.sh:20]: done

**Comment by raviprak-altiscale** [comment]
+1. LGTM

**Pull request merged**